### DISCUSSION TITLE
What is the maximum streaming time ?

### DISCUSSION THREAD

**Discussion by AtoLrn** [remix-run/remix#8109] [Q&A]
Hi,

I am currently using the streaming response with remix with an around 60s loader.  I can't get the data to get to the client.
I tried with a timeout of 15s and in this case everything works well, but when the streaming takes more time, I randomly never gets an answer.

I set a abort delay of 2 minutes in the entry,server,ts but the request is never resolving.

Also in the network I see the initial page load randomly finishing between 15s and 50s even if the call is still going on the server. Do anyone have an idea of what happens ? 

I am using remix 2.2.0 btw :)

Thanks

**Comment by sergiodxa** [comment]
Remix itself doesn't set a maximum streaming time, so this can be your browser closing the connection or the HTTP server closing the connection or maybe a CDN closing it.

What are you reading that takes 60s? Is there no other way to get that data? Maybe cache it somewhere and read from cache? Or are you using the loader and defer to run a background process?
